core/src/test/scala/com/nvidia/spark/rapids/tool/tuning/ProfilingAutoTunerSuite.scala

```diff
@@ -834,130 +834,6 @@ class ProfilingAutoTunerSuite extends BaseAutoTunerSuite {
     compareOutput(expectedResults, autoTunerOutput)
   }
 
-  test("test T4 dataproc cluster with dynamicAllocation enabled") {
```

Comment:
These are legacy [tests](https://github.com/NVIDIA/spark-rapids/blob/branch-22.12/tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/AutoTunerSuite.scala#L370) that were based on a previous logic of not recommending `spark.executor.instances` when dynamic allocation was enabled ([logic](https://github.com/NVIDIA/spark-rapids/blob/branch-22.12/tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala#L421))

However, in [PR #1188](https://github.com/NVIDIA/spark-rapids-tools/pull/1188) we started recommending `spark.executor.instances` even when dynamic allocation is enabled, which makes these UTs outdated.